### PR TITLE
Add language spec to code blocks

### PR DESCRIPTION
**Pull request by hi-rustin**
1. Added language spec to code blocks to have better syntax support.
2. In Markdown, the angle brackets (<>) are used to automatically convert a URL into a clickable link. So my linter added it.

[rendered](https://github.com/hi-rustin/crates.io/blob/rustin-patch-code-block/docs/CONTRIBUTING.md)

### CODE REVIEW

**Comment by hi-rustin** [comment]
The comparison:

Before:
image
image

After:

image

image

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ This will install a binary named `diesel`, so you should be able to run `diesel
 --version` to confirm successful installation.
 
 > If you're on Linux and this fails with an error that looks like `error:
-linking with `cc` failed: exit code: 1`, you're probably missing some
+linking with`cc`failed: exit code: 1`, you're probably missing some
```
Comment (Turbo87) [inline docs/CONTRIBUTING.md:296]: why did you remove the spaces?

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -194,11 +194,11 @@ methods we'd recommend for each operating system:
...
-  >     ```
-  >     error: linking with `cc` failed: exit code: 1
-  >     [lots of output]
-  >     = note: /usr/bin/ld: cannot find -l[something]
-  >     ```
+  >```console
+  >error: linking with `cc` failed: exit code: 1
+  >[lots of output]
+  >= note: /usr/bin/ld: cannot find -l[something]
+  >```
```
Comment (Turbo87) [inline docs/CONTRIBUTING.md:201]: This code block is text output. My understanding is that console code blocks are specifically for commands that you enter into the console, no?

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (hi-rustin) [reply docs/CONTRIBUTING.md:296]: Good catch. I think the linter thought it was `a` and `b`, but it actually is `a and b`. I will fix it.

Comment (hi-rustin) [reply docs/CONTRIBUTING.md]: Updated.

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (hi-rustin) [reply docs/CONTRIBUTING.md:201]: I think the console also contains what is printed out in your console(stdout + stderr). But I've updated it.

**Review by hi-rustin — COMMENTED** [review]
🔢 Self-check

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -136,8 +136,8 @@ talk to:
...
-| `pnpm start:staging`                         | https://staging-crates-io.herokuapp.com                           | Testing UI changes with a smaller set of realistic data |
+| `pnpm start:live`                            | <https://crates.io>                                                 | Testing UI changes with the full live site's data       |
+| `pnpm start:staging`                         | <https://staging-crates-io.herokuapp.com>                           | Testing UI changes with a smaller set of realistic data |
 | `pnpm start`                                 | Static fixture test data in `mirage/fixtures`                     | Setting up particular situations, see note              |
 | `pnpm start:local`                           | Backend server running locally                                    | See the Working on the backend section for setup        |
 | `PROXY_BACKEND=https://crates.io pnpm start` | Whatever is specified in the `PROXY_BACKEND` environment variable | If your use case is not covered here                    |
```
Comment (Marcono1234) [inline docs/CONTRIBUTING.md:143]: Should the indentation here then be adjusted? The last column of these rows is shifted compared to the other rows:
Column shifted in diff

Comment (hi-rustin) [reply docs/CONTRIBUTING.md]: Good catch! Updated.

**Comment by hi-rustin** [comment]
@Turbo87 Ping~ Could you please take another look when you have time? Thanks!

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (Turbo87) [reply docs/CONTRIBUTING.md]: you can escape backticks by doubling the number of outer backticks.

for example: 

```
``foo`bar`baz``
```

will render as ``foo`bar`baz``

I've added a small commit on top that adjusts this :)
